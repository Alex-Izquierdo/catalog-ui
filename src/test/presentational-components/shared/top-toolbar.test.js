import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import TopToolbar from '../../../presentational-components/shared/top-toolbar';

describe('<TopToolbar />', () => {
  it('should render correctly', () => {
    expect(shallowToJson(shallow(
      <TopToolbar>
        <div>Children</div>
      </TopToolbar>
    ))).toMatchSnapshot();
  });
});
