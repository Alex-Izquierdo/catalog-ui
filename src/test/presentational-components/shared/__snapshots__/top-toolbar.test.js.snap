// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopToolbar /> should render correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "white",
      "marginLeft": 0,
      "marginRight": 0,
      "marginTop": 0,
      "paddingLeft": 24,
      "paddingRight": 24,
      "paddingTop": 48,
    }
  }
>
  <styled.div
    className="pf-u-pt-lg pf-u-pr-lg pf-u-pl-lg pf-u-pb-lg"
  >
    <div
      className="pf-u-mb-md"
    >
       
      <CatalogBreadcrumbs
        breadcrumbfragments={Array []}
      />
    </div>
    <div>
      Children
    </div>
  </styled.div>
</div>
`;
